Sources/Public/CalendarView.swift

```diff
@@ -455,16 +455,20 @@ public final class CalendarView: UIView {
   }
 
   private func targetAnchorLayoutItem(for scrollToItemContext: ScrollToItemContext) -> LayoutItem {
+    let offset = CGPoint(
+      x: scrollView.contentOffset.x + directionalLayoutMargins.leading,
+      y: scrollView.contentOffset.y + directionalLayoutMargins.top)
```

Comment:
similar logic in https://github.com/airbnb/HorizonCalendar/pull/63/files#diff-cb95d9bcbe378d78650b19323f50a949L443 - I just forgot to implement it in both places a few PRs ago when I first added layout margins support.